reviews/admin.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-from django.contrib import admin
+# from django.contrib import admin
```

Comment:
Why is this commented out?

Reply:
Because flake8 complained, and I hesitated to delete the file.

Reply:
Oh I get it, it's the only thing in the file.